nixos/modules/services/networking/firewall.nix

```diff
@@ -278,13 +278,35 @@ in
         '';
       };
 
+      defaultInterfaces = lib.mkOption {
```

Comment:
I don't like using an empty list to mean something other than an empty list; `defaultInterfaces = "all"` would be the ideal UX, (I think you have to write the type as `either (enum [ "all" ]) (listOf str)`). Using `null` would be good too.